### PR TITLE
docs: add service to readme

### LINKED ISSUE #991: docs: improve readme
Our current readme, seems to be a little bit outdated. This project has evolved significantly and it's no longer just "CLI & SDK" like we describe it in readme. I think we should restructure the readme to cover all different components of this project and different aspects of usage:

* renku-project
    1. [core] library and how to use it from Python as SDK
    2. [cli] This is covered mostly in our docs, so providing the link to those onboarding tutorials
    3. [svc] Service documentation is covered through OpenAPI spec and we should put a link to it (either swagger UI from the dev deployment or just link to a json file)

I propose we think about how we can restructure our readme so it provides better on boarding experience for new users and cover different types of users in a better way.

### PR DESCRIPTION
**Pull request by rokroskar**
closes #991

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -15,9 +15,9 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 
-==============================
- Renku CLI and SDK for Python
-==============================
+=================================
+ Renku Python library and Service
```
Comment (jsam) [inline README.rst:19]: What happened to CLI?

Comment (rokroskar) [reply README.rst:19]: I was trying to not be too verbose. 

"Renku Python Library, CLI, and Service"?

File: README.rst [hunk 2]
```diff
@@ -185,8 +200,8 @@ still possible with ``pipx``. First clone the repository and then do:
 This will install all the extras for testing and debugging.
 
 
-Run tests
----------
+Runing tests
```
Comment (Panaetius) [inline README.rst:203]:
```suggestion
Running tests
```

File: README.rst [hunk 3]
```diff
@@ -169,11 +174,21 @@ your data analysis workflows. The full documentation will soon be available
...
+for the `RenkuLab <https://renkulab.io>`_ web UI. The service can be deployed in
+production as a Helm chart (see `helm-chart <./helm-chart/README.rst>`_.
+
+
 Developing Renku
 ================
 
-For development it's convenient to install ``renku`` in editable mode. This is
-still possible with ``pipx``. First clone the repository and then do:
+For testing the funtionality from source it is convenient to install ``renku``
```
Comment (Panaetius) [inline README.rst:190]:
```suggestion
For testing the functionality from source it is convenient to install ``renku``
```

**Pull request merged**